content/pt/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html

```diff
@@ -29,16 +29,16 @@ <h3>Objetivos</h3>
             <div class="col-md-8">
                 <h3>Clusters do Kubernetes</h3>
                 <p>
-                <b>O Kubernetes coordena um cluster altamente disponível de computadores conectados para funcionar como uma única unidade.</b>
-                As abstrações no Kubernetes permitem implantar aplicativos em contêineres em um cluster sem amarrá-los especificamente a máquinas individuais.
-                Para fazer uso desse novo modelo de implantação, os aplicativos precisam ser empacotados de uma forma que os desacople dos hosts individuais: eles precisam ser colocados em contêineres. Os aplicativos em contêineres são mais flexíveis e disponíveis do que nos modelos de implantação anteriores, nos quais os aplicativos eram instalados diretamente em máquinas específicas como pacotes profundamente integrados ao host. 
+                <b>O Kubernetes coordena um cluster com alta disponibilidade de computadores conectados para funcionar como uma única unidade.</b>
+                As abstrações no Kubernetes permitem implantar aplicativos em contêineres em um cluster sem amarrá-los especificamente as máquinas individuais.
+                Para fazer uso desse novo modelo de implantação, os aplicativos precisam ser empacotados de uma forma que os desacoplem dos hosts individuais: eles precisam ser empacotados em contêineres. Os aplicativos em contêineres são mais flexíveis e disponíveis do que nos modelos de implantação anteriores, nos quais os aplicativos eram instalados diretamente em máquinas específicas como pacotes profundamente integrados ao host. 
                 <b> O Kubernetes automatiza a distribuição e o agendamento de contêineres de aplicativos em um cluster de maneira mais eficiente. </b> 
                 O Kubernetes é uma plataforma de código aberto e está pronto para produção.
                 </p>
                 <p>Um cluster Kubernetes consiste em dois tipos de recursos:
                     <ul>
-                        <li>O <b>Master</b> coordena o cluster </li>
-                        <li>Os <b>Nodes</b> são os trabalhadores que executam aplicativos</li>
+                        <li>A <b>Camada de gerenciamento <i>(Control Plane)</i></b> coordena o cluster </li>
```

Comment:
IMO we could call "Control Plane" "Plano de controle". Even google is using this term in their [translations](https://cloud.google.com/anthos/clusters/docs/aws/concepts/architecture?hl=pt-br#management_cluster).
Camada de Gerenciamento looks like to me that we have a layer over another or something like that.
But from the K8s [docs](https://d33wubrfki0l68.cloudfront.net/2475489eaf20163ec0f54ddc1d92aa8d4c87c96b/e7c81/images/docs/components-of-kubernetes.svg) its more like a group of softwares that have a purpose.

What do you think @jailton and @rikatz?

Reply:
To be really honest, I've hard feelings about this. Some of Google translations are automated, and I guess some things actually don't make sense to the end users. I've seen some authors translating slices to "fatias" where this should be kept as slices :)

We've discussed that some words make more sense staying in English (for Portuguese translations) and putting them in Italic. I don't think this is the case, BUT "Plano de Controle" to me represents less than "Camada de gerenciamento" :)

On a past translation I did, I was discussing with @jcjesus about the proper translation and we've reached the consensus of going with "Camada de Gerenciamento" due to a lack of a better translation :)

But this is my opinion, would like to listen other folks

Reply:
Hi, @devlware. I'll send a message on our channel to discuss this issue.